filecoin-proofs/Cargo.toml

```diff
@@ -47,6 +47,12 @@ rand_xorshift = "0.2.0"
 typenum = "1.11.2"
 bitintr = "0.3.0"
 gperftools = { version = "0.2", optional = true }
+phase2 = { package = "phase21", git = "https://github.com/filecoin-project/phase2", rev = "dc3e6f5" }
+# phase2 = { package = "phase21", path = "/mnt/storage/phase2" }
+# phase2 = { package = "phase21", version = "..."}
```

Comment:
Change this when new phase2 release is published.

Reply:
published phase21@0.4